src/components/Main/Modal/FileModal/FileModalContentFooter.vue

```diff
@@ -33,18 +35,27 @@ export default defineComponent({
   },
   setup(props, context) {
     const { fileMeta } = useFileMeta(props, context)
-
-    let channelPath = ''
-    try {
-      channelPath = useChannelPath().channelIdToPathString(
-        fileMeta.value?.channelId ?? '',
-        true
-      )
-    } catch {}
     const user = store.state.entities.users[fileMeta.value?.uploaderId ?? '']
-    const userName = '@' + user?.name
     const createdAt = getCreatedDate(fileMeta.value?.createdAt ?? '')
-    return { channelPath, userName, createdAt }
+    const { channelIdToPathString } = useChannelPath()
+    const channelPath = computed(() => {
+      try {
+        return fileMeta.value?.channelId
+          ? channelIdToPathString(fileMeta.value?.channelId)
+          : ''
+      } catch {
+        return ''
+      }
+    })
+
+    const onClick = async () => {
+      if (!fileMeta.value?.channelId) return
```

Comment:
```suggestion
      if (channelPath.value === '') return
```
こっちじゃないと動かないことがあるのでこっちで:pray: